### PR TITLE
Note app pool restarts with the bundle installer

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #8428 

Thanks @bernhof for surfacing this. :rocket:

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -172,6 +172,8 @@ Enable the **IIS Management Console** and **World Wide Web Services**.
 
 1. Restart the system or execute **net stop was /y** followed by **net start w3svc** from a command prompt. Restarting IIS picks up a change to the system PATH, which is an environment variable, made by the installer.
 
+   If an ASP.NET Core Module installer detects that IIS requires a reset in order to complete installation, the installer resets IIS. If the installer triggers an IIS reset, all of the IIS app pools are restarted.
```
[the remark this answers is not in the document]
Comment (jkotalik) [reply aspnetcore/host-and-deploy/iis/index.md]: @shirhatti is investigating if the installers call iisreset.exe or net start/stop was so you can be more precise.

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: @shirhatti Did u get a chance to look into this yet?

> If an ASP.NET Core Module installer detects that IIS requires a reset in order to complete installation, the installer resets IIS. If the installer triggers an IIS reset, all of the IIS app pools are restarted.

The question is: Is the IIS reset performed via *iisreset.exe* or `net stop/start was`?

[the remark this answers is not in the document]
Comment (guardrex) [reply aspnetcore/host-and-deploy/iis/index.md]: @jkotalik Is the installer in a repo somewhere? ... I can go take a look. I'm trying to merge stuff before my PTO lands next week.

**Comment by Rick-Anderson** [comment]
@shirhatti please review - it's 1 line only.

**Comment by jkotalik** [comment]
He is oof today. LGTM though.

**Comment by guardrex** [comment]
Fixes #8428

**Pull request merged**